app/models/journal_entry.rb

```diff
@@ -0,0 +1,42 @@
+class JournalEntry < ActiveRecord::Base
```

Comment:
Would a better name for this be "RotaGenerationLog" as it is tightly bound to that.
JournalEntry is pretty generic .. ?